_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -117,45 +123,253 @@ This `button` element has the text "search" rendered as an magnifying glass icon
 <button aria-label="Find">search</button>
 ```
 
+#### Passed Example 7 
+
+This button has [visible inner text][] that, according to the [label in name algorithm][], is contained within the [accessible name][].  This example shows why the [label in name algorithm][] uses the [visible inner text][] and not the [visible text content][]: the <p> tags insert whitespace into the result in the former but not the latter.
+
+```html
+<button aria-label="Hello world"><p>Hello</p><p>world</p></button>
+```
+
+#### Passed Example 8
+
+Similar to the previous example.
+
+```html
+<a href="#" aria-label="Some article by John Doe"><h6>Some article</h6><p>by John Doe</p></a>
+```
+
+#### Passed Example 9
+
+The [visible inner text][] of this link is "ACT" (with no spaces) because of the explicit styles of "display: inline" on the &lt;p&gt; elements and the absence of whitespace between the &lt;p&gt; elements.  The cases of "display: inline" and "display: block" are handled by the definition of [visible inner text of an element][].  This example shows that the definition agrees with the visual rendering done by the browser.
```

Comment:
```suggestion
The [visible inner text][] of this link is "ACT" (with no spaces) because of the explicit styles of `display: inline` on the `p` elements and the absence of whitespace between the `p` elements.  The cases of `display: inline` and `display: block` are handled by the definition of [visible inner text of an element][].  This example shows that the definition agrees with the visual rendering done by the browser.
```

Reply:
Done in 2dcd941d3a853e5dcee6acfc0332ac9de1b72515.